lib/agent.js

```diff
@@ -93,6 +96,27 @@ class Agent {
         }
     }
 
+    async getCurrentPackage () {
+        if (this.launcher) {
+            return await this.launcher.readPackage()
+        }
+        return Promise.resolve(null)
+    }
+
+    async getCurrentFlows () {
+        if (this.launcher) {
+            return await this.launcher.readFlow()
+        }
+        return Promise.resolve(null)
+    }
+
+    async getCurrentCredentials () {
+        if (this.launcher) {
+            return await this.launcher.readCredentials()
+        }
+        return Promise.resolve(null)
```

Comment:
```suggestion
        return null
    }

    async getCurrentFlows () {
        if (this.launcher) {
            return await this.launcher.readFlow()
        }
        return null
    }

    async getCurrentCredentials () {
        if (this.launcher) {
            return await this.launcher.readCredentials()
        }
        return null
```

You do not need to return `Promise.resolve(...)` in an async function - the fact the function is async means there is an implicit promise wrapped around anything that is returned.